client/package.json

```diff
@@ -45,7 +45,7 @@
     "start:ui-stage": "sh -ac '. .env.staging; react-app-rewired start'",
     "start": "run-p -l api:dev api:doc start:css start:ui",
     "start:stage": "open https://localhost:5001/login/; npm run build:clean; run-p -l api:stage start:css start:ui-stage",
-    "start:prod": "npm run api:prod; open https://localhost/login/",
+    "start:prod": "npm run api:prod; open https://localhost:9443/login/",
```

Comment:
I am uncertain about if this one needed to be changed or not.

Reply:
I am pretty sure this one needs to be changed